modules/static/gogo/protobuf/initref

```diff
@@ -1 +1 @@
-f6b4bb7b2dde1736b809b3da996ed72f278e9be9
+v0.3
```

Comment:
This is the closest tag:

```
➜  protobuf git:(f6b4bb7b) git describe --all f6b4bb7b2dde1736b809b3da996ed72f278e9be9
tags/v0.3-140-gf6b4bb7b
```


Here are all the releases: https://github.com/gogo/protobuf/releases